frontend/src/screens/wallet/swap/SwapStatus.tsx

```diff
@@ -0,0 +1,545 @@
+                          {!swap.claimTxId &&
+                            " You can use the Process Refund button to claim the locked up bitcoin."}
+                        </TooltipContent>
+                      </Tooltip>
+                    </TooltipProvider>
+                  </div>
+                  <Divider color="border-red-500" />
+                </>
+              )}
+              {swapStatus === "PENDING" && (
+                <>
+                  <div className="flex items-center text-muted-foreground text-sm">
+                    <Loading className="w-5 h-5 mr-2" />
+                    Confirming lockup deposit...
+                  </div>
+                  <Divider color="border-green-600 dark:border-emerald-500" />
+                  <div className="flex items-center text-muted-foreground text-sm">
+                    <CircleCheckIcon className="w-5 h-5 mr-2 text-green-600 dark:text-emerald-500" />
+                    <div className="flex items-center gap-2">
+                      <p>Lockup found in mempool.</p>
+                      <ExternalLink
+                        to={`https://mempool.space/tx/${swap.lockupTxId}`}
+                        className="flex items-center underline text-foreground"
+                      >
+                        View
+                      </ExternalLink>
+                    </div>
+                  </div>
+                  <Divider color="border-green-600 dark:border-emerald-500" />
+                </>
+              )}
+              <div className="flex items-center text-muted-foreground text-sm">
+                <CircleCheckIcon className="w-5 h-5 mr-2 text-green-600 dark:text-emerald-500" />
+                Swap initiated
+              </div>
+            </div>
+          )}
+        </CardContent>
+      </Card>
+    </div>
+  );
+}
+
+function SwapOutStatus({ swap }: { swap: SwapOut }) {
+  const { toast } = useToast();
+  const { isDarkMode } = useTheme();
+
+  const copyTxId = () => {
+    copyToClipboard(swap.claimTxId as string, toast);
+  };
+
+  const defaultOptions = {
+    loop: true,
+    autoplay: true,
+    animationData: isDarkMode ? animationDataDark : animationDataLight,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
+  const swapStatus = swap.state;
+  const statusText = {
+    SUCCESS: "Swap Successful",
+    FAILED: "Swap Failed",
+    PENDING: swap.lockupTxId
+      ? "Waiting for lockup confirmation"
+      : "Waiting for deposit from boltz",
+  };
+
+  return (
+    <div className="w-full max-w-lg">
+      <Card className="w-full md:max-w-xs">
+        <CardHeader>
+          <CardTitle className="flex justify-center">
+            {swapStatus === "PENDING" && <Loading className="w-4 h-4 mr-2" />}
+            {statusText[swapStatus]}
+          </CardTitle>
+        </CardHeader>
+        <CardContent className="flex flex-col items-center gap-4">
+          {swapStatus === "SUCCESS" && (
```

Comment:
@im-adithya I think the swap out should only be fully successful once there is a confirmation. The transaction is only broadcasted, not confirmed.

![image](https://github.com/user-attachments/assets/ec6ee999-44e6-4d6a-833c-ba0a1cf13bbd)

Reply:
Also I think we are way too technical and do not need all these steps / states. Who apart from you will understand it as it is now?